webpack.config.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const path = require('path');
+
+/**@type {import('webpack').Configuration}*/
+const config = {
+    target: 'node', // vscode extensions run in a Node.js-context 📖 -> https://webpack.js.org/configuration/node/
+    entry: './src/extension.ts', // the entry point of this extension, 📖 -> https://webpack.js.org/configuration/entry-context/
+    output: {
+        // the bundle is stored in the 'dist' folder (check package.json), 📖 -> https://webpack.js.org/configuration/output/
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'extension.js',
```

Comment:
How about using a more detailed name?

Reply:
This is the name suggested by VSCode official document.
https://code.visualstudio.com/api/working-with-extensions/bundling-extension#configure-webpack

Reply:
I think that's just a sample rather than a suggested name?

Reply:
@style95 
Yes, this is not a suggested name. But many VSCode extensions use extension.js or extension.ts as the entry point(main point), and I think they have been affected by documentation.

I don't have a good idea for the detailed name actually 😂 
Could you recommend the name?

### Ref
- https://github.com/microsoft/vscode-chrome-debug/blob/master/package.json#L78
- https://github.com/DonJayamanne/gitHistoryVSCode/blob/master/webpack.config.js#L72
- https://github.com/eamodio/vscode-gitlens/blob/master/webpack.config.js#L74

Reply:
How about simply using `openwhisk-extension.js` or `vscode-openwhisk-extension.js`?